src/index.html

```diff
@@ -3,6 +3,7 @@
 <meta charset="utf-8">
 <meta name="description" content="Gain Qlik Core data model insights quickly during its development and validation phases.">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link href="//fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
What icons are we using? We removed the material icons library earlier this week..